fe/fe-core/src/main/java/org/apache/doris/nereids/rules/implementation/AggregateStrategies.java

```diff
@@ -734,24 +734,24 @@ inputToBufferParam, maybeUsingStreamAgg(connectContext, logicalAgg),
      *
      * after:
      *
-     *  single node aggregate:
+     *  anyLocalGatherGlobalAgg:
```

Comment:
the first agg must after distribution. our normal count function do not have the ability of distinct. the distinct complished by group by [name, id]. so we must shuffle all tuples with same name into one partition, and then do group by [name, id] to get unique id list of same name.